### PR TITLE
Allow fetching MTP by specific block height

### PR DESCRIPTION
**Pull request by lukachi**
To obtain data by a specific block in the Rarimo state contract and generate proof for a specific state, an optional parameter `blockHeight` can now be added to the `loadDataFromRarimoCore` method in the snap package.

This resolves the potential issue of an invalid proof signature when the signed proof conflicts with the requested core and target state details, which may cause them to be unmatched.

### CODE REVIEW

File: packages/snap/src/config.ts [hunk 1]
```diff
@@ -59,4 +60,14 @@ export const SUPPORTED_CHAINS: Record<number, ChainInfo> = {
...
+  [CHAINS.Sepolia]: {
+    id: CHAINS.Sepolia,
+    rpcUrl: 'https://endpoints.omniatech.io/v1/eth/sepolia/public',
+    stateContractAddress: '0x8a9F505bD8a22BF09b0c19F65C17426cd33f3912',
+  },
+  [CHAINS.Goerly]: {
+    id: CHAINS.Goerly,
+    rpcUrl: 'https://ethereum-goerli.publicnode.com',
+    stateContractAddress: '0x0F08e8EA245E63F2090Bf3fF3772402Da9c047ee',
+  },
```
Comment (Electr1Xx) [inline packages/snap/src/config.ts:72]: remove testnet chains

Comment (ihordiachenko) [reply packages/snap/src/config.ts:72]: We've decided to keep them

**Comment by ihordiachenko** [comment]
@lukachi update the changelog to reflect the changes

File: packages/connector/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rarimo/rarime-connector",
-  "version": "0.2.0",
+  "version": "0.3.0",
```
Comment (ihordiachenko) [inline packages/connector/package.json:3]: Does this change extend the public API/UI of the snap and the connector in any way? If no, this should a patch version according to semver. So the version should be `0.2.1`

**Review by ihordiachenko — APPROVED** [review]

**Pull request merged**